src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1291,9 +1291,20 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateAppVrHelp(
     return NULL;
   }
   smart_objects::SmartObject& vr_help = *result;
-  vr_help[strings::vr_help_title] = app->name();
+  const smart_objects::SmartObject* vr_help_title = app->vr_help_title();
+  if (NULL != vr_help_title &&
```

Comment:
@ValeriiMalkov use `nullptr` instead of `NULL` or even `if(ptr)`

Reply:
@AByzhynar That was fixed at the commit: 0f55c360d24a552e95417775277ee1d62c39ed43